flink-connectors/flink-connector-hbase/src/test/java/org/apache/flink/connector/hbase/source/hbaseendpoint/HBaseEndpointTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.hbase.source.hbaseendpoint;
+
+import org.apache.flink.connector.hbase.source.TestsWithTestHBaseCluster;
+import org.apache.flink.connector.hbase.source.reader.HBaseEvent;
+import org.apache.flink.connector.hbase.testutil.HBaseTestClusterUtil;
+
+import org.apache.hadoop.hbase.Cell;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/** Tests for {@link HBaseEndpoint}. */
+public class HBaseEndpointTest extends TestsWithTestHBaseCluster {
+
+    @Test
+    public void testSetup() throws Exception {
```

Comment:
No assertions?

Reply:
Yes. Tests the setup. Test should pass if no error occurs.

Reply:
Close: "Expect no to throw" is a JUnit 5 feature